org.civicrm.bootstrapcivihr/scss/pages/reports.scss

```diff
@@ -1,4 +1,4 @@
-body[class*="page-civihr-reports"] {
+.page-civihr-reports {
 
     .form-inline {
```

Comment:
@AkA84 If this is not usable anymore then, we will remove this file as we discussed.